### DISCUSSION TITLE
Do we need to embed metadata in the markdown prose sections?

### DISCUSSION THREAD

**Discussion by rprimet** [CatalaLang/catleg#19] [General]
For instance, to add directives for `catdev` such as exemptions from linting (like `noqa` in flake), or to indicate how to process images when they have been manually converted to latex formulas etc.

If so, how? Should we use code fences with a specific identifier (e.g. \`\`\`catdev)?

**Comment by denismerigoux** [comment]
Issue related to https://github.com/CatalaLang/catala/issues/392. I don't have any strong opinions on this. @AltGr ?
